static/app/views/performance/transactionSummary/transactionOverview/tagExplorer.spec.jsx

```diff
@@ -255,7 +236,7 @@ describe('WrapperComponent', function () {
     );
   });
 
-  it('Check sort links of headers', async function () {
+  it('renders the table headers in the correct order', async function () {
```

Comment:
I had to change what this test does, since it wasn't straightforward to convert the behaviour of the test. I tried interactively clicking on the page headers via `userEvent` to confirm the sorting behaviour, but the headers are links so it appears that the component is not actually updating